index.html

```diff
@@ -0,0 +1,203 @@
+                <article class="project3">
+                    <div class="wrap-card">
+                        <h3>Website Protfolio</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore quia aspernatur exercitationem quo libero?</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
+                    </div>
+                    <div class="card-link">
+                        <a class="see-project" href="#">See Project</a>
+                    </div>
+                </article>  
+                <article class="project4">
+                    <div class="wrap-card">
+                        <h3>Profesional Art Printing Data More</h3>
+                        <p>Quod cupiditate, minima, asperiores architecto deleniti ullam tenetur doloremque earum nihil quam, maxime quo enim et!</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
+                    </div>
+                    <div class="card-link">
+                        <a class="see-project" href="#">See Project</a>
+                    </div>
+                </article>
+                <article class="project5">
+                    <div class="wrap-card">
+                        <h3>Data Dashboard Healthcare</h3>
+                        <p> Tempore quia aspernatur exercitationem quo libero? Quod cupiditate, minima, asperiores architecto deleniti ullam tenetur doloremque earum nihil quam, maxime quo enim et!</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
+                    </div>
+                    <div class="card-link">
+                        <a class="see-project" href="#">See Project</a>
+                    </div>
+                </article>
+                 <article class="project6">
+                    <div class="wrap-card">
+                        <h3>project 6</h3>
+                        <p> Tempore quia aspernatur exercitationem quo libero? Quod cupiditate, minima, asperiores architecto deleniti ullam tenetur doloremque earum nihil quam, maxime quo enim et!</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
```

Comment:
- [ ] [REQUIRED] The list of project technologies should(e.g., "JavasScript", "HTML" and "CSS") a list `ul` and each of them a `li`. Please make changes!